src/vs/workbench/contrib/terminal/browser/terminalService.ts

```diff
@@ -971,6 +981,29 @@ export class TerminalService implements ITerminalService {
 		return this._createTerminal(shellLaunchConfig, location, options);
 	}
 
+	async createDetachedXterm(options: IDetachedXTermOptions): Promise<IXtermTerminal> {
+		const ctor = await TerminalInstance.getXtermConstructor(this._keybindingService, this._contextKeyService);
+		const instance = this._instantiationService.createInstance(
+			XtermTerminal,
+			ctor,
+			this._configHelper,
+			options.cols,
+			options.rows,
+			{
+				getBackgroundColor: () => Color.transparent,
+			},
```

Comment:
Transparent will probably cause some problems, you should implement this similar to in terminal instance but with the right values for all the places this particular detached instance can appear:

https://github.com/microsoft/vscode/blob/b7fd221daf9c426b4f0ca126e70b3a1d58d15d91/src/vs/workbench/contrib/terminal/browser/terminalInstance.ts#L724-L737